example/iframe.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <title>Secure Wallet Generation</title>
     <meta charset="utf-8" />
-    <script src="./dist/src/index.js"></script>
+    <script src="./dist/wallet-generator.js"></script>
```

Comment:
@saraswatpuneet  This breaks `npm run dev`, but not sure what the fix is for it.

Reply:
i see, i reverted this and modified ci to generate same pattten